bindinfo/bind_test.go

```diff
@@ -496,6 +496,17 @@ func (s *testSuite) TestCapturePlanBaseline(c *C) {
 	c.Assert(len(rows), Equals, 1)
 	c.Assert(rows[0][0], Equals, "select * from t where a > ?")
 	c.Assert(rows[0][1], Equals, "SELECT /*+ USE_INDEX(@`sel_1` `test`.`t` )*/ * FROM `t` WHERE `a`>10")
+
+	// Capture plan baseline on invisible index.
+	tk.MustExec("create table t1(a int, unique (a) invisible)")
+	tk.MustExec("insert into t1 values (1), (2)")
+	tk.MustQuery("select a from t1 order by a").Check(testkit.Rows("1", "2"))
+	tk.MustQuery("select a from t1 order by a").Check(testkit.Rows("1", "2"))
+	tk.MustExec("admin capture bindings")
+	rows = tk.MustQuery("show global bindings").Rows()
+	c.Assert(len(rows), Equals, 2)
+	c.Assert(rows[1][0], Equals, "select a from t1 order by a")
+	c.Assert(rows[1][1], Equals, "SELECT /*+ USE_INDEX(@`sel_1` `test`.`t1` )*/ `a` FROM `t1` ORDER BY `a`")
```

Comment:
No, I mean, create a binding using index, then make the index invisible.

Reply:
ditto